src/ui/bottomsheet/bottomsheet.ts

```diff
@@ -0,0 +1,92 @@
+import { IEventEmitter } from '../../core/eventemitter';
+import { INativeMobileComponent, MobileOSProps } from '../../core/native-mobile-component';
+import { IView } from '../view/view';
+import { BottomSheetEvents } from './bottomsheet-events';
+
+/**
+ * @class UI.BottomSheet
+ * @since 5.0.4
+ * @android
+ *
+ * BottomSheet represents a bottom sheet. A bottom sheet helps people perform a scoped task that’s closely related to their current context. 
+ * This component only supported on Android.
+ *      
+ *     @example
+ *     ```
+ *     const content = new FlexLayout();
+ *     content.height = 300;
+ *     content.width = 300;
+ *     const bottomSheet = new BottomSheet({
+ *        view: content,
+ *        borderRadius: 10,
+ *        detents: detents,
+ *     });
+ *     bottomSheet.on('dismissed', () => {
+ *       console.log("on dismissed, isShowing: ", bottomSheet.isShowing);
+ *     })
+ *     bottomSheet.show();
+ *     ```
+ */
+export interface IBottomSheet<TEvent extends string = BottomSheetEvents> extends INativeMobileComponent<any, MobileOSProps>, IEventEmitter<TEvent | BottomSheetEvents> {
+
+  /**
+   * Determines the view of the bottom sheet.
+   *
+   * @property {UI.IView} view
+   * @android
+   * @since 5.0.4
+   */
+  view: IView | undefined;
```

Comment:
Is this `IView | undefined` because view is optional or can it be also given undefined to not show anything?